packages/driver/cypress/integration/cypress/script_utils_spec.js

```diff
@@ -42,10 +43,18 @@ describe('src/cypress/script_utils', () => {
     it('evals each script', () => {
       return $scriptUtils.runScripts(scriptWindow, scripts)
       .then(() => {
-        expect($sourceMapUtils.extractSourceMap).to.be.calledTwice
-        expect($sourceMapUtils.extractSourceMap).to.be.calledWith(scripts[0], 'the script contents')
-        expect($sourceMapUtils.extractSourceMap).to.be.calledWith(scripts[1], 'the script contents')
+        expect(scriptWindow.eval).to.be.calledTwice
+        expect(scriptWindow.eval).to.be.calledWith(Cypress.sinon.match(/^the script contents/))
```

Comment:
Regex hurts my soul. Can we find a better matcher? Perhaps

```suggestion
        expect(scriptWindow.eval).to.be.alwaysCalledWith('the script contents')
```

Reply:
Ah, perhaps I don't understand what eval is supposed to do :-), it's unclear from the matcher what the object type is. I honestly prefer the verbose way as it's more clear.

Reply:
An alternative could be to just put the entire message - that might make it even *more* clear.

Reply:
The test you present here is the same exact test as the `extracts the source map from each script` test.

Reply:
I replaced the regexp with the complete string

Reply:
Yeah. Just wanted to ace the regex 👍🏻